scalafmt-tests/src/test/resources/align/DefaultWithAlign.stat

```diff
@@ -1850,7 +1850,7 @@ case class Foo() {
 case class Foo() {
   def Foo(firstParam: Int, secondParam: String, thirdParam: Boolean) = ???
   def Bar(abc:        Int, deg:         String, xyz:        Boolean) = ???
-  def Baz(hijk:       Int, lmnop:       String) = ???
+  def Baz(hijk:       Int, lmnop:       String)                      = ???
```

Comment:
Isn't this a regression?

Reply:
no, this is the whole point. try to align some nested columns (each line has different number) but then align the top stuff the usual way.